README.md

```diff
@@ -62,3 +62,9 @@ To watch specific test(s) run `npm t -- -w -g "exchangeRates get` or even
 To get the code coverage with Istanbul/nyc execute : `npm run test-cov`. You should see the code coverage on the cli.
 
 You can also check the code coverage on [code climate](https://codeclimate.com/github/geshan/currency-api/src/exchangeRates.js/source).
+
+### Husky hooks - Prettier
+
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+
+Prettier is an opinionated code formatter. It enforces a consistent style by parsing your code and re-printing it with its own rules that take the maximum line length into account, wrapping code when necessary.
```

Comment:
don't copy paste, explain how it works :). Fix the commit message to be imperative.